### DISCUSSION TITLE
AV1 support for WebRTC streaming

### DISCUSSION THREAD

**Discussion by vec715** [sipsorcery-org/sipsorcery#1050] [Q&A] [answered]
I am interested in using AV1 as an encoder for video streaming. H.264 is not patent-free, which makes it unsuitable for commercial use. VP8, on the other hand, has a very low frame rate (around 8-13 FPS), which is not ideal for my needs. AV1 appears to be a promising solution, as it is both performant and free to use. However, when I tried to use the example provided in the GitHub repository (https://github.com/sipsorcery-org/sipsorcery/tree/master/examples/WebRTCExamples/FfmpegToWebRTC), it seemed that AV1 support was not yet implemented?

**Comment by sipsorcery** [comment, marked as answer]
AV1 support hasn't been added to the sipsorcery library. The only reason is no one has gotten around to it yet. PR's welcome.

Why do you say VP8 has a limted frame rate? The codec itself doesn't limit the frame rate and WebRTC doesn't apply any such restriction.
